### PR TITLE
PUBDEV-8091 nicer api for baseline hazard and survival

### PR DESCRIPTION
**Pull request by satai**

### CODE REVIEW

**Comment by satai** [comment]
Does this one look OK to you, @grigory93 ?

**Review by grigory93 — APPROVED** [review]
thanks!

**Comment by grigory93** [comment]
documentation question - do new accessors get documented automatically?
we should have our Cox PH documentation example(s) expanded with new functionality too...

**Comment by michalkurka** [comment]
@satai thank you - LGTM - but please rebase on top of rel-zipf and change the base branch

**Comment by satai** [comment]
> @satai thank you - LGTM - but please rebase on top of rel-zipf and change the base branch

Zipfed, @michalkurka

**Review by michalkurka — CHANGES_REQUESTED** [review]
@satai you are changing generated classes, the change needs to be made in the helpers that the generator uses

take a look at eg. gen_aggregator.py and method class_extensions: https://github.com/h2oai/h2o-3/blob/master/h2o-bindings/bin/custom/python/gen_aggregator.py#L4

File: h2o-py/h2o/estimators/coxph.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import h2o
```
Comment (michalkurka) [inline h2o-py/h2o/estimators/coxph.py:9]: you don't need this import if you use H2OFrame.get_frame

**Comment by michalkurka** [comment]
@hannah-tillman please be aware of this PR when working on PUBDEV-8091

**Comment by michalkurka** [comment]
@satai @grigory93 I fixed the issues, change the naming to conform to h2o conventions, please take a look

**Review by michalkurka — APPROVED** [review]

**Pull request merged**